tests/utils_tests.py

```diff
@@ -1344,3 +1345,18 @@ def test_log_this(self) -> None:
             json.loads(record.json)["form_data"]["viz_type"],
             slc.viz.form_data["viz_type"],
         )
+
+
+def test_parse_table_full_name():
+    assert parse_table_full_name("table") == (None, "table")
+    assert parse_table_full_name("table", schema="schema") == ("schema", "table")
+    assert parse_table_full_name("table_schema.table", schema="schema") == (
+        "table_schema",
+        "table",
+    )
+    assert parse_table_full_name("table_schema.table") == ("table_schema", "table")
+    assert parse_table_full_name("hive.table_schema.table") == ("table_schema", "table")
+    assert parse_table_full_name("nonsense.hive.table_schema.table") == (
+        None,
+        "nonsense.hive.table_schema.table",
+    )
```

Comment:
For the sake of uniformity we should probably use `self.assertEqual` here?

Reply:
pytest style is using assert instead of self.assertEqual: https://docs.pytest.org/en/stable/assert.html

Reply:
Ok, got it! 👍